restapi/stories.go

```diff
@@ -57,12 +73,74 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 
 	for i := 0; i < len(stories); i++ {
 		city := stories[i].CurrentCity
-		coordinates, err := api.locationFinder.GetCityCenter(city)
+		stories[i].CurrentCity = strings.Title(strings.ToLower(city))
+
+		// get number of stories with current city in db
+		var prevStories []models.Story
```

Comment:
I think in gorm you can use `.count` (or something similar) to get the count. I think we should use that as loading all stories in memory (even if its a single city) could cause us issues. [https://stackoverflow.com/questions/54720000/how-display-result-count-from-query](https://stackoverflow.com/questions/54720000/how-display-result-count-from-query)

Reply:
okay used count